lib/chartlib.sh

```diff
@@ -26,8 +26,11 @@ readonly TIMEOUT="${TIMEOUT:-300}"
 readonly LINT_CONF="${LINT_CONF:-/testing/etc/lintconf.yaml}"
 readonly CHART_YAML_SCHEMA="${CHART_YAML_SCHEMA:-/testing/etc/chart_schema.yaml}"
 readonly VALIDATE_MAINTAINERS="${VALIDATE_MAINTAINERS:-true}"
-readonly CHECK_VERSION_INCREMENT="${CHECK_VERSION_INCREMENT:-true}"
 readonly GITHUB_INSTANCE="${GITHUB_INSTANCE:-https://github.com}"
+# env vars below aren't readonly to allow us to overwrite them
+CHECK_VERSION_INCREMENT="${CHECK_VERSION_INCREMENT:-true}"
+CHART_ALL="${CHART_ALL:-false}"
+CHART="${CHART:-}"
```

Comment:
`CHART` and `CHART_ALL` should not be here. The script does not need them.

`CHECK_VERSION_INCREMENT` should be `readonly`.

Reply:
`CHECK_VERSION_INCREMENT` cannot be readonly as otherwise `chart_test.sh` cannot overwrite it, see the error below:
`chart_test.sh: line 123: CHECK_VERSION_INCREMENT: readonly variable`

Reply:
and for simplicity we do not want to use env var from cli.

Reply:
> so we keeping only flags option, no configuration via env vars?

I see now what you mean. I found this to be a little messy and inconsistent.

> `CHECK_VERSION_INCREMENT` cannot be readonly as otherwise

It cannot be readonly as it is now. It needs to be exported before `chart_lib.sh` is sourced.

Reply:
ok, only flags are left, that was my intention in the beginning:)

Added extra `if` with `CHECK_VERSION_INCREMENT=false` that will allow to have it as readonly.